spec/requests/secure_cookies_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+COOKIE_SEPARATOR = "\n".freeze
+
+RSpec.describe 'secure cookies' do
+  it 'flags all cookies sent by the application as secure, httponly, and lax' do
+    get root_url
+    cookie_count = response.headers['Set-Cookie'].split(COOKIE_SEPARATOR).count
```

Comment:
we shouldn't define this constant here, because it "leaks" into the global namespace across all files in the project

A few options:
Reference the constant from the class:
```suggestion
RSpec.describe 'secure cookies' do
  it 'flags all cookies sent by the application as secure, httponly, and lax' do
    get root_url
    cookie_count = response.headers['Set-Cookie'].split(SecureCookies::COOKIE_SEPARATOR).count
```

Just inline it

```suggestion
RSpec.describe 'secure cookies' do
  it 'flags all cookies sent by the application as secure, httponly, and lax' do
    get root_url
    cookie_count = response.headers['Set-Cookie'].split("\n").count
```